diff --git a/android/src/main/kotlin/com/shopify/reactnative/flash_list/BlankAreaEvent.kt b/android/src/main/kotlin/com/shopify/reactnative/flash_list/BlankAreaEvent.kt
index ed74c43582af56b6dc541460d91d1ede53156d9e..dd34130a6f0be48ddecb3e322b1614013b46140e 100644
--- a/android/src/main/kotlin/com/shopify/reactnative/flash_list/BlankAreaEvent.kt	
+++ b/android/src/main/kotlin/com/shopify/reactnative/flash_list/BlankAreaEvent.kt
@@ -20,7 +20,7 @@ class BlankAreaEvent(
     }
 
     override fun dispatch(rctEventEmitter: RCTEventEmitter) {
-        rctEventEmitter.receiveEvent(viewTag, eventName, eventData)
+        rctEventEmitter.receiveEvent(viewTag, eventName, getEventData())
     }
 
     companion object {
